yarn.lock

```diff
@@ -12279,7 +12281,7 @@ resolve@^1.1.4, resolve@^1.10.0, resolve@^1.14.2, resolve@^1.15.1, resolve@^1.17
     path-parse "^1.0.7"
     supports-preserve-symlinks-flag "^1.0.0"
 
-resolve@^1.10.1:
+resolve@^1.10.1, resolve@^1.22.1:
```

Comment:
There was another version of resolve used in the project. We could unify versions if we keep using resolve instead of is-core-module.